platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/recovery/EventRecoveryWorkflow.java

```diff
@@ -124,30 +126,36 @@ public static void recoverState(
 
         logger.info(STARTUP.getMarker(), "Loading state from {}", signedStateFile);
 
-        final SignedState initialState =
-                SignedStateFileReader.readStateFile(signedStateFile).signedState();
+        final PlatformContext platformContext =
+                new DefaultPlatformContext(configuration, new NoOpMetrics(), CryptographyHolder.get());
+
+        final ReservedSignedState initialState = SignedStateFileReader.readStateFile(platformContext, signedStateFile)
```

Comment:
It doesnt matter much for this tool, but this reserved state should be closed.

Reply:
Good catch. I agree, we should do it right.